scargo/commands/flash.py

```diff
@@ -51,6 +63,8 @@ def flash_esp32(
                 "--partition-name=spiffs",
                 f"--input={fs_path}",
             ]
+            if port:
+                command.append(f"--port={port}")
             subprocess.check_call(command, cwd=project_path)
         else:
             command = ["esptool.py", "--chip", target.id, "write_flash", "@flash_args"]
```

Comment:
`esptool.py`, according to documentation, has `--port` option that functions the same as in `parttool.py`. Is there a reason you don't use it here?